app.rb

```diff
@@ -0,0 +1,64 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './classroom'
+
+class App
+  attr_accessor :people, :books, :rentals, :classroom
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @classroom = 'Default Class'
+  end
+
+  def list_books
+    if @books.length.positive?
+      books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+    else
+      puts 'No books added.'
+    end
+  end
+
+  def list_people
+    if @people.length.positive?
+      people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    else
+      puts 'No people added.'
+    end
+  end
+
+  def create_student(age, name, parent_permission)
+    student = Student.new(age, name, @classroom, parent_permission: parent_permission)
+    @people.push(student)
+    puts 'Student created successfully'
```

Comment:
- [ ] Your app is breaking in attempt to create a person of type student. Kindly ensure to improve upon this:

**Error creating student:**
![image](https://user-images.githubusercontent.com/12745474/188191127-cfb1eddd-23de-4b04-a202-671797844887.png)